cxplat/inc/cxplat/common.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation
+// SPDX-License-Identifier: MIT
+
+#pragma once
+#include <assert.h>
+#include <sal.h>
+
+#ifdef CXPLAT_SOURCE
+#define CXPLAT_API __declspec(dllexport)
+#else
+#define CXPLAT_API __declspec(dllimport)
+#endif
+
+#ifdef _DEBUG
+#define cxplat_assert(x) assert(x)
```

Comment:
For MsQuic, we generally follow the pattern that `#define` macros are always all upper case: `CXPLAT_ASSERT`. Additionally, we have several types of asserts:
```c
#define CXPLAT_DBG_ASSERT(_exp) 
#define CXPLAT_DBG_ASSERTMSG(_exp, _msg)
#define CXPLAT_TEL_ASSERT(_exp)
#define CXPLAT_TEL_ASSERTMSG(_exp, _msg)
#define CXPLAT_TEL_ASSERTMSG_ARGS(_exp, _msg, _origin, _bucketArg1, _bucketArg2)
#define CXPLAT_FRE_ASSERT(_exp)
#define CXPLAT_FRE_ASSERTMSG(_exp, _msg)
`

Reply:
_FRE_ is super confusing and not cross-platform as a term.  I propose using `_RUNTIME_`

Reply:
I'm definitely open to other terms. Not sure how I feel about `RUNTIME` though.

Reply:
I'm open to other terms too :)